schema/partner_artist.js

```diff
@@ -0,0 +1,72 @@
+import gravity from '../lib/loaders/gravity';
+import Partner from './partner';
+import Artist from './artist/index';
+import numeral from './fields/numeral';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLBoolean,
+} from 'graphql';
+
+const PartnerArtistType = new GraphQLObjectType({
+  name: 'PartnerArtist',
+  fields: () => ({
+
+    counts: {
+      type: new GraphQLObjectType({
+        name: 'PartnerArtistCounts',
+        fields: {
+          artworks: numeral(({ published_artworks_count }) =>
+            published_artworks_count),
+          for_sale_artworks: numeral(({ published_for_sale_artworks_count }) =>
+            published_for_sale_artworks_count),
+        },
+      }),
+      resolve: (partner_artist) => partner_artist,
+    },
+    id: {
+      type: GraphQLString,
+    },
+    is_display_on_partner_profile: {
+      type: GraphQLBoolean,
```

Comment:
Don't these need a `resolve` with `display_on_partner_profile`? That's the JSON that comes back from gravity.